services/token_pause.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2018 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+option java_package = "com.hederahashgraph.api.proto.java";
+option java_multiple_files = true;
+
+import "basic_types.proto";
+
+/**
+ * Pauses the Token from being involved in any kind of Transaction until it is unPaused.
+ * Must be signed with Token's PauseKey.
+ * If the provided token is not found, the transaction will resolve to INVALID_TOKEN_ID.
+ * If the provided token has been deleted, the transaction will resolve to TOKEN_WAS_DELETED.
+ * If no Pause Key is defined, the transaction will resolve to TOKEN_HAS_NO_PAUSE_KEY.
+ * Once executed the Token is marked as paused and will be not able to be a part of any transaction.
+ */
+message TokenPauseTransactionBody {
+  /**
+   * The token to be paused. If token does not exist, transaction
+   * results in INVALID_TOKEN_ID
```

Comment:
It seems redundant to document here again which response codes will be returned when it's already above.
```suggestion
   * The token to be paused.
```